desktop-exporter/app/routes/trace-view.jsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
-import { useParams, Link } from 'react-router-dom';
+import { useParams, Link, useLoaderData } from 'react-router-dom';
+
+export async function traceLoader({ params }) {
+    const response = await fetch(`/traces/${params.traceID}`);
+    const traceData = await response.json();
+    return traceData;
+}
 
 
 export default function TraceView() {
-    let { traceID } = useParams();
+    const { spans } = useLoaderData();
+    console.log(spans);
     return (
         <>
             <Link to={"/"}>Back</Link>
-            <p>I am Item {traceID}</p>
+            <p>I am Item {spans[0].traceID}</p>
```

Comment:
We should probably add `TraceID` to `TraceData` soon https://github.com/CtrlSpice/otel-desktop-viewer/blob/main/desktop-exporter/types.go#L22